beep/features/featurizer_helpers.py

```diff
@@ -16,7 +16,7 @@
 from scipy.optimize import curve_fit
 
 
-def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, max_nr_peaks, rpt_type, half_peak_width=0.075):
+def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, cwt_range, max_nr_peaks, rpt_type, half_peak_width=0.075):
```

Comment:
What does 'cwt' stand for? Can we make the parameter name `cwt_range`  a little more explanatory?

Reply:
It is the range of the scaling parameter for the continuous wavelet transform (CWT) algorithm that is used in the peak finding function.

Reply:
Got it, if you can mention it in the docstring that would be great